### PR TITLE
Feat/appointment request

### PR DESCRIPTION
**Pull request by ProggerOleg**
## Describe your changes

- Created drawer for appointment-request + modal window to display seekers health questionnaire.
- Added B&F to update status of appointment on button click.
- Changed header to make Info button optional
Screenshot from 2023-12-10 12-24-25

Diagnoses/Challenges modal window:
Screenshot from 2023-12-11 15-55-29

Activities modal window:
Screenshot from 2023-12-12 16-55-59

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://homecareaid.atlassian.net/jira/software/projects/CC/boards/1?issueType=10001&selectedIssue=CC-123)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [x] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/pages/profile/index.tsx [hunk 1]
```diff
@@ -82,7 +82,16 @@ function Profile(): JSX.Element | null {
...
+<<<<<<< HEAD
+      <FlowHeader
+        text={translate('profile.headerTitle')}
+        iconType="back"
+        infoButton
+        callback={handleBack}
+      />
+=======
       <FlowHeader text={translate('profile.headerTitle')} />
+>>>>>>> develop
```
Comment (Simpsonkk) [inline src/pages/profile/index.tsx:94]: You forgot to remove conflicts things

Comment (ProggerOleg) [reply src/pages/profile/index.tsx:94]: you're right, fixed

**Review by fedasevich — APPROVED** [review]

File: src/components/health-questionnaire/constants.ts [hunk 2]
```diff
@@ -1,3 +1,9 @@
+export const Steps = [
```
Comment (lyudad) [inline src/components/health-questionnaire/constants.ts:1]: why it uppercase

File: src/components/health-questionnaire/constants.ts [hunk 3]
```diff
@@ -1,3 +1,9 @@
...
+  'health_questionnaire.diagnoses',
+  'health_questionnaire.activity',
+  'health_questionnaire.env',
+];
+
 export const Options = [
```
Comment (lyudad) [inline src/components/health-questionnaire/constants.ts:7]: why it uppercase

Comment (ProggerOleg) [reply src/components/health-questionnaire/constants.ts]: fixed

File: src/components/health-questionnaire/constants.ts (hunk 2 above)
Comment (ProggerOleg) [reply src/components/health-questionnaire/constants.ts]: fixed

**Review by lyudad — APPROVED** [review]

**Pull request merged**